<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>418</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>200</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>niu.png</normaloff>niu.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="Button_connect">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="Button_export">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>导出ID</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="Button_exportdata">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>导出数据</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>资产号段</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>模块ID结束号码：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" colspan="2">
        <widget class="QLineEdit" name="Edit_MidEnd">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="Edit_MidStart">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>模块ID起始号码：</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="MyQComboBox" name="cBox_pc_filter">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>选择批次</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionset"/>
    <addaction name="actionexit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionset">
   <property name="text">
    <string>数据库</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>筛选</string>
   </property>
   <property name="toolTip">
    <string>筛选</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyQComboBox</class>
   <extends>QComboBox</extends>
   <header>myqcombobox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
